### ISSUE TITLE
Anti-endomysium antibody positivity

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6604]
**Preferred term label:** Anti-endomysial antibody (EmA) positivity

**Synonyms**

**Definition (free text, please give PubMed ID)**
Autoantibodies directed against endomysial tissue transglutaminase  2 (tTG2).  PMID 9518950

**Parent term (use hpo.jax.org/app)**
Anti-transglutaminase antibody positivity (once created)

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Celiac Disease OMIM:212750

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik 
So I take it that the target of this is not known, but suspected to be tissue transglutaminase?
https://pmj.bmj.com/content/76/898/466
Would that mean they should be made synonyms?

**Comment by eswietlik** [comment]
@pnrobinson I think the garget is always TG but depending if localisation the Abs are called either EMA or anti-epidermal TG Ab.

**Comment by pnrobinson** [comment]
@eswietlik so do you mean
(parent) Anti-tissue transglutaminase antibody positivity
------(child) Anti-endomysial antibody (EmA) positivity
------(child) Anti-epidermal transglutaminase antibody positivity

**Comment by pnrobinson** [comment]
added
